frontend/src/components/Navigation/Navigation.tsx

```diff
@@ -0,0 +1,38 @@
+import { useLocation } from "react-router-dom"
+import * as NavigationMenu from "@radix-ui/react-navigation-menu"
+import styles from "./Navigation.module.css"
+
+const navigationRoutes = [{ name: "Dashboard", href: "/dashboard" }]
+
+const Navigation = () => {
+  const location = useLocation()
+
+  return (
+    <NavigationMenu.Root className={styles["navigation"]}>
+      <NavigationMenu.List className={styles["navigation-menu-list"]}>
+        {navigationRoutes.map((navigationRoute, i) => {
+          return (
+            <NavigationMenu.Item key={i}>
```

Comment:
Would it be possible and reasonable to use `navigationRoute.name` as key instead of index?